albumentations/augmentations/functional.py

```diff
@@ -740,14 +740,14 @@ def add_sun_flare(
 
 @ensure_contiguous
 @preserve_shape
-def add_shadow(img: np.ndarray, vertices_list: List[List[Tuple[int, int]]]) -> np.ndarray:
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `add_shadow`

Reason for update: The type of 'vertices_list' argument has changed from a list of lists of tuples to a list of numpy.ndarray.

Suggested new docstring:
```
"""Add shadows to the image.

From https://github.com/UjjwalSaxena/Automold--Road-Augmentation-Library

Args:
    img (numpy.ndarray):
    vertices_list (list[numpy.ndarray]):

Returns:
    numpy.ndarray:
"""
```